bin/check-rabbitmq-consumer-utilisation.rb

```diff
@@ -0,0 +1,167 @@
+#!/usr/bin/env ruby
+
+# Check RabbitMQ consumer utilisation
+# ===
+#
+# DESCRIPTION:
+# This plugin checks the consumer utilisation percentage.
+# The fraction of time in which the queue is able to immediately deliver
+# messages to consumer. If this number drops in percentage this may result
+# in slower message delivery and indicate issues with the queue.
+#
+# Seeded from check-rabbitmq-consumers.rb
+#
+# PLATFORMS:
+#   Linux, BSD, Solaris
+#
+# DEPENDENCIES:
+#   RabbitMQ rabbitmq_management plugin
+#   gem: sensu-plugin
+#   gem: carrot-top
+#
+# LICENSE:
+# Copyright 2014 Daniel Kerwin <d.kerwin@gini.net>
+# Copyright 2014 Tim Smith <tim@cozy.co>
+# Copyright 2018 Mike Murray <37150283+monkey670@users.noreply.github.com>
+#
+# Released under the same terms as Sensu (the MIT license); see LICENSE
+# for details.
+
+require 'sensu-plugin/check/cli'
+require 'carrot-top'
+require 'inifile'
+
+# main plugin class
+class CheckRabbitMQConsumerUtilisation < Sensu::Plugin::Check::CLI
+  option :host,
```

Comment:
rather than re-defining these again lets pull them in from [here](https://github.com/sensu-plugins/sensu-plugins-rabbitmq/blob/5.0.0/lib/sensu-plugins-rabbitmq/rabbitmq.rb#L10) by depending on our own base check cli class for rabbitmq like [this](https://github.com/sensu-plugins/sensu-plugins-rabbitmq/blob/5.0.0/bin/check-rabbitmq-queues-synchronised.rb#L28).